packages/react-accessible-tooltip/src/Tooltip.js

```diff
@@ -61,20 +72,37 @@ class Tooltip extends Component<TooltipProps, TooltipState> {
         } else if (!currentTarget.contains(newTarget)) {
             this.hide();
         }
-    }
+    };
+
+    // This handles the support for touch devices that do not trigger blur on 'touch-away'.
+    handleTouch = ({ target }: Event) => {
+        const { activeElement } = document;
+
+        if (!(activeElement instanceof Element) || !(target instanceof Element))
```

Comment:
In `acc-css-app` we're testing for `instanceof HTMLElement`, but this would not catch `HTMLBodyElement` for example. `Element` is a bit of a broader net.

Reply:
Aren't we going to apply this as well at line `70`?

Reply:
We could, but it's a bit of a moot point because it's not possible for a `label` component to be an HTMLBodyElement.

Reply:
Yup. Fair enough.